src/components/body/body.component.ts

```diff
@@ -187,12 +198,15 @@ export class DataTableBodyComponent implements OnInit, OnDestroy {
 
   rowHeightsCache: RowHeightCache = new RowHeightCache();
   temp: any[] = [];
+  temp2: any[] = [];
```

Comment:
I'm not sure why you are looking at an older change. I should really figure out how to rebase my WIP commits. temp2 was used in the above template (data-selection rows input property) that requires direct raw access to the row data. If you look at https://github.com/swimlane/ngx-datatable/pull/737/files#diff-955af42ecfa71378797f9996448e885aR352 you can see that I map it when it's available. I tried to propagate changes to other components where I could.